common/procedures.go

```diff
@@ -20,20 +21,21 @@ const (
 )
 
 var procStrMap = map[ProcedureType]string{
-	REGISTRATION_PROCEDURE:                   "REGISTRATION-PROCEDURE",
-	PDU_SESSION_ESTABLISHMENT_PROCEDURE:      "PDU-SESSION-ESTABLISHMENT-PROCEDURE",
-	USER_DATA_PKT_GENERATION_PROCEDURE:       "USER-DATA-PACKET-GENERATION-PROCEDURE",
-	UE_INITIATED_DEREGISTRATION_PROCEDURE:    "UE-INITIATED-DEREGISTRATION-PROCEDURE",
-	AN_RELEASE_PROCEDURE:                     "AN-RELEASE-PROCEDURE",
-	UE_TRIGGERED_SERVICE_REQUEST_PROCEDURE:   "UE-TRIGGERED-SERVICE-REQUEST-PROCEDURE",
-	NW_TRIGGERED_UE_DEREGISTRATION_PROCEDURE: "NW-TRIGGERED-UE-DEREGISTRATION-PROCEDURE",
-	AMF_RELEASE_PROCEDURE:                    "AMF_RELEASE_PROCEDURE",
+	REGISTRATION_PROCEDURE:                     "REGISTRATION-PROCEDURE",
+	PDU_SESSION_ESTABLISHMENT_PROCEDURE:        "PDU-SESSION-ESTABLISHMENT-PROCEDURE",
+	USER_DATA_PKT_GENERATION_PROCEDURE:         "USER-DATA-PACKET-GENERATION-PROCEDURE",
+	UE_INITIATED_DEREGISTRATION_PROCEDURE:      "UE-INITIATED-DEREGISTRATION-PROCEDURE",
+	AN_RELEASE_PROCEDURE:                       "AN-RELEASE-PROCEDURE",
+	UE_TRIGGERED_SERVICE_REQUEST_PROCEDURE:     "UE-TRIGGERED-SERVICE-REQUEST-PROCEDURE",
+	NW_TRIGGERED_UE_DEREGISTRATION_PROCEDURE:   "NW-TRIGGERED-UE-DEREGISTRATION-PROCEDURE",
+	AMF_RELEASE_PROCEDURE:                      "AMF_RELEASE_PROCEDURE",
```

Comment:
Could you please update "AMF_RELEASE_PROCEDURE" as well ? instead of _ , use - ?